trace86/jhu-aai
# Add data generation

Added data generation category and made it easier to implement or remove delays and verbosity.

To test:
* Check out this branch `add_data_generation`
* Run `init_game.py`
* Make sure it runs
